cypress/support/commands.js

```diff
@@ -23,3 +23,22 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+    cy.get(`[placeholder="${placeholder}"]`)
+  });
+
+Cypress.Commands.add('login', (username, password) => {
+    cy.request({
```

Comment:
Use the provided 'username' and 'password' parameters instead of hardcoded values for the login request body 📝